byzcoin/transaction.go

```diff
@@ -345,11 +345,12 @@ func (instrs Instructions) Hash() []byte {
 type TxResults []TxResult
 
 // NewTxResults takes a list of client transactions and wraps them up
-// in a TxResults with Accepted set to false for each.
+// in a TxResults with Accepted set to true for each.
 func NewTxResults(ct ...ClientTransaction) TxResults {
 	out := make([]TxResult, len(ct))
 	for i := range ct {
 		out[i].ClientTransaction = ct[i]
+		out[i].Accepted = true
```

Comment:
Why does this need to be true? It seems to be more natural to default these flags to false because the moment that `NewTxResults` is used is before we verify the transactions `createStateChanges`.

Reply:
as written above: can't remember anymore, and it doesn't change anything.